client/src/client.rs

```diff
@@ -213,13 +291,59 @@ impl Client {
     ) -> Result<SBOMUploadResponse> {
         let url = Client::create_upload_url(cloud_front_domain, team_id, project_id, codebase_id);
 
-        let response: Option<SBOMUploadResponse> =
-            post(url.as_str(), sbom_token, Some(sbom)).await?;
+        println!("Using token {}, Uploading SBOM to: {}", sbom_token, url);
+
+        let response: Option<SBOMUploadResponse> = post(
+            url.as_str(),
+            ContentType::Json,
+            sbom_token,
+            Some(sbom)
+        ).await?;
 
         Ok(response.unwrap())
     }
 }
 
+/// Helper method that creates a client and uploads sbom based on provided information
+pub async fn simple_upload_sbom(cloud_front_domain: String, sbom_token: String, team_id: String, orig_url: String, language: String, sbom: HashMap<String, Value>) {
```

Comment:
I like the idea here.  This is not a blocker at all, but we need to look at this client package vs. a HarborService in the CLI. I think it there might be some dependency cycles to work out though.

Reply:
I agree, this is something we should work out with @qtpeters as its possible we can have a lot more reusable code.